ci/docker-compose-build.yaml

```diff
@@ -19,16 +19,11 @@ services:
     image: tcf-ci-build:${ISOLATION_ID}
 
     build:
-      context: .
-      dockerfile: ../docker/Dockerfile.tcf-dev
+      context: ..
+      dockerfile: docker/Dockerfile.tcf-dev
       args:
         - http_proxy
         - https_proxy
         - no_proxy
     volumes:
       - ../:/project/TrustedComputeFramework
```

Comment:
Is volume mounting required after this change?

Reply:
Not needed, will remove it